source/extensions/common/wasm/context.cc

```diff
@@ -722,6 +725,16 @@ WasmResult Context::getHeaderMapValue(WasmHeaderMapType type, absl::string_view
                                       absl::string_view* value) {
   auto map = getConstMap(type);
   if (!map) {
+    if (access_log_phase_) {
+      // Maps might point to nullptr in the access log phase.
+      if (wasm()->abiVersion() == proxy_wasm::AbiVersion::ProxyWasm_0_1_0) {
```

Comment:
Is it necessary to return Ok on 0.1.0?

Reply:
Yes, see the happy path a few lines below.

Reply:
I see. Thanks.